commit 5491867479e658227b14b61466d349eb22f45693 (refs/changes/56/1156/2)
Author: David Pacheco <dap@joyent.com>
Date:   2016-12-21T12:42:13-08:00 (2 years, 10 months ago)
    
    MANTA-3061 storage zone DNS configuration leads to binder overload in large configs
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/sapi_manifests/registrar/template b/sapi_manifests/registrar/template
index 8fdfaa2..52136ad 100644
--- a/sapi_manifests/registrar/template
+++ b/sapi_manifests/registrar/template
@@ -2,7 +2,7 @@
 	"registration": {
 		"domain": "{{SERVICE_NAME}}",
 		"type": "host",
-		"ttl": 60,
+		"ttl": {{#REGISTRAR_TTL}}{{REGISTRAR_TTL}}{{/REGISTRAR_TTL}}{{^REGISTRAR_TTL}}1800{{/REGISTRAR_TTL}},
 		"aliases": [
 			"{{MANTA_STORAGE_ID}}"
 		]
